react/OrderPlaced.js

```diff
@@ -1,39 +1,55 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
+import { compose, graphql } from 'react-apollo'
 import Header from './components/Header/Header'
 import OrderInfo from './components/OrderInfo/OrderInfo'
-import { orderGroupQuery } from './test-cases/split-order-2-sellers'
 import { profileQuery } from './test-cases/profileQuery'
+import getOrderGroup from './graphql/getOrderGroup.graphql'
+import withNoSSR from './withNoSSR'
 
 export const CurrencyContext = React.createContext('BRL')
 export const SplitOrderContext = React.createContext(false)
 class OrderPlaced extends Component {
   render() {
-    return (
-      <CurrencyContext.Provider value={orderGroupQuery.orderGroup[0].storePreferencesData.currencyCode}>
-        <SplitOrderContext.Provider value={orderGroupQuery.orderGroup.length > 1}>
-          <div className="center">
-            <Header data={orderGroupQuery.orderGroup} profile={profileQuery.profile} />
-            {
-              orderGroupQuery.orderGroup.map(order => {
-                return (
-                  <OrderInfo
-                    data={order}
-                    profile={profileQuery.profile}
-                    key={order.orderId}
-                  />
-                )
-              })
-            }
-          </div>
-        </SplitOrderContext.Provider>
-      </CurrencyContext.Provider>
-    )
+    const { orderGroupQuery } = this.props
+    return (orderGroupQuery.loading) ? null
+      : (
+        <CurrencyContext.Provider value={orderGroupQuery.orderGroup[0].storePreferencesData.currencyCode}>
+          <SplitOrderContext.Provider value={orderGroupQuery.orderGroup.length > 1}>
+            <div className="center">
```

Comment:
Useful?

Reply:
This conditional will be removed. It was necessary because if the page were to be displayed before the `getOrderGroup` query was completed, I would receive an error.

I'll remove this and use `branch()` from `recompose` to prevent getting that same error in the next commit.

Reply:
Updated :)